homeassistant/components/microsoft_face/__init__.py

```diff
@@ -174,13 +179,27 @@ async def async_delete_person(service):
     async def async_face_person(service):
         """Add a new face picture to a person."""
         g_id = service.data[ATTR_GROUP]
-        p_id = face.store[g_id].get(service.data[ATTR_PERSON])
+        p_name = service.data[ATTR_PERSON]
+
+        if g_id not in face.store:
+            msg = "Can't add an image of the person '{}' with error: Unknown group '{}'".format(
```

Comment:
Please use f-strings instead of str.format.

Reply:
Ok. Just pushed format -> f-strings.